components/datetime/src/date.rs

```diff
@@ -48,14 +48,22 @@ pub trait DateTimeType: FromStr {
 /// Temporary implementation of [`DateTimeType`],
 /// which is used in tests, benchmarks and examples of this component.
 ///
+/// *Notice:* Rust at the moment does not have a canonical way to represent date and time. We are introducing
+/// `MockDateTime` as an example of the data necessary for ICU [`DateTimeFormat`] to work, and
+/// we hope to work with the community to develop core date and time APIs that will work as an input for this component.
```

Comment:
Can you add a link to https://github.com/unicode-org/icu4x/blob/master/docs/research/date_time.md ?

Reply:
Sure, that helps with the clarity here. I also added links to the other locations with similar text.